tools/vmd/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+
+exports_files(["ios_vm_test_runner.template.sh"])
+
+sh_binary(
+    name = "vmd",
+    srcs = ["vm_run.sh"],
+    data = [
+        # Note: there are a plethora of problems from doing a trival build of this
+        # depenency with Bazel. For now, vendor the binary (56kb)
+        #
+        # We could always make a certificate - this needs further auditing for
+        # production usage.
+        "bin/sshpass",
+        "@tart//file",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+sh_binary(
+    name = "ios_vm_test_runner",
+    srcs = ["ios_vm_test_runner.sh"],
+    data = [
+        ## We want to call the original runner under vm_run
+        "vmd",
+        "@xctestrunner//:ios_test_runner",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+# Wrap everything into a CLI
+pkg_tar(
+    name = "vmd_tar",
+    srcs = [
+        "bin/sshpass",
+        "bin/tart",
+        ":vmd",
+    ],
+)
```

Comment:
Why do we need an empty WORKSPACE in this folder?

Reply:
Good catch